sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -82,7 +86,54 @@ public void setExtra(String key, String value) {
   }
 
   @Override
-  protected Scope clone() throws CloneNotSupportedException {
-    return (Scope) super.clone();
+  public Scope clone() throws CloneNotSupportedException {
+    Scope clone = (Scope) super.clone();
+    clone.level = level != null ? SentryLevel.valueOf(level.name().toUpperCase(Locale.ROOT)) : null;
+    clone.user = user != null ? user.clone() : null;
+    clone.fingerprint = fingerprint != null ? new ArrayList<>(fingerprint) : null;
+
+    if (breadcrumbs != null) {
+      Queue<Breadcrumb> breadcrumbsClone =
+          SynchronizedQueue.synchronizedQueue(new CircularFifoQueue<>(maxBreadcrumb));
```

Comment:
Can we have this line as a private method? 
The intent is to have a single place where we define what concrete types are involved in creating breadcrumb list.
Right now it's here and in the constructor.